src/Utility/Server.ts

```diff
@@ -31,11 +31,17 @@ export default class Server {
             log: false,
         });
 
-        return server.init({...this.browsersyncOptions, ...{
-            middleware: [
-                dev,
-                hmr,
-            ],
-        }});
+        this.browsersyncOptions.forEach((options) => {
+            const server = browsersync.create(name).init({...options, ...{
+                middleware: [
+                    dev,
+                    hmr,
+                ],
```

Comment:
I think sharing the middlewares with all server instances here, is the correct thing to do instead of creating seperate middleware instances for each server instance; I hope.